.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ orbs:
   # Pins Helm to v2.x
   # TODO: Consider upgrading Helm to v3.0 (https://github.com/StackStorm/stackstorm-ha/issues/98)
```

Comment:
```suggestion
  # TODO: Consider upgrading Helm to v3.0 (https://github.com/StackStorm/stackstorm-ha/issues/98)
```
this comment could be removed now

Reply:
I have removed the comment and finished the circleci upgrade to helm v3